modules/dw1000/dw1000.c

```diff
@@ -250,7 +258,7 @@ static void dw1000_config(struct dw1000_instance_s* instance) {
         // [0x1000:0x1001] LDE_PPINDX
         // [0x1002:0x1003] LDE_PPAMPL
         // [0x1804:0x1805] LDE_RXANTD
-        dw1000_write16(instance, 0x2E, 0x1804, 21620);
+        dw1000_write16(instance, 0x2E, 0x1804, config.ant_delay);
```

Comment:
Should we just write a constant to the antenna delay and then correct the timestamps on our own?